src/cartridges/bm_adyen/cartridge/static/default/adyenSettings.js

```diff
@@ -86,6 +90,76 @@ document.addEventListener('DOMContentLoaded', () => {
     this.classList.toggle('bi-eye');
   }
 
+  // open Adyen Giving Background upload page
+  function uploadAdyenGivingBackground() {
+    const params = 'resizable=no,width=1000,height=500,left=100,top=100';
+    const openedWindow = window.open(
+      window.adyenGivingBackgroundUrl,
+      'test',
```

Comment:
Why is this called test?

Reply:
It is the popup window name, forgot to change it.
Will update!